transport.go

```diff
@@ -15,6 +15,7 @@ import (
 	"compress/flate"
 	"compress/gzip"
 	"compress/zlib"
+	"compress/zstd"
```

Comment:
This is not a valid package as far as I can see.
Replacing it with `github.com/klauspost/compress/zstd` is fine and compatible.

Reply:
@Dissociable thank you